src/interiorpointsolver.jl

```diff
@@ -292,7 +292,7 @@ function eval_jac_wrapper!(ipp::Solver, kkt::AbstractKKTSystem, x::Vector{Float6
     ns = length(ipp.ind_ineq)
     @trace(ipp.logger, "Evaluating constraint Jacobian.")
     jac = get_jacobian(kkt)
-    cnt.eval_function_time += @elapsed nlp.con_jac!(jac, view(x,1:nlp.n))
+    cnt.eval_function_time += @elapsed jac_coord!(nlp,view(x,1:get_nvar(nlp)),jac)
```

Comment:
I don't know how this will play with `DenseKKTSystem`

Reply:
For dense, we have a different callback function `jac_dense!`, but this can be changed after coordinating with JSO folks